cmd/definitions/tmpl/service.tmpl

```diff
@@ -306,6 +303,11 @@ func (s *{{ $pn }})parsePair{{ $pn }}{{ $fnk }}(opts []Pair) (pair{{ $pn }}{{ $f
             err = s.formatError("{{$fn.Name}}", err {{ $path }} )
         }()
 
+        {{- template "mode_check" makeSlice $fn.ObjectMode $fn.Params.ObjectParamName }}
```

Comment:
How about expanding the template here? It adds an extra burden to understand the logic here (introduce a new template for just three lines).

Reply:
And I think we don't need to check whether we have Object here.

Those functions that have object mode must have object as param.

Check `$fn.ObjectMode` is enough to me.

Reply:
The `ObjectParamName` is for getting the param name of `Object`, since we can ensure the name of param `Object` is always `o`, maybe I should remove this func?

Reply:
Yes, Object will always be `o *Object`, it's ensured by our specs.